bedrock/mozorg/tests/test_helper_misc.py

```diff
@@ -988,49 +988,6 @@ def test_monitor_fxa_button(self):
         self.assertEqual(markup, expected)
 
 
-@override_settings(FXA_ENDPOINT=TEST_FXA_ENDPOINT)
```

Comment:
Suggestion: we still have the `relay_fxa_button` helper, and might well use it again in the future, so I'd suggest keeping this test?

Reply:
I initially removed the test because we no longer have an instance of the button, and it seemed to be checking for "expected markup" and I assumed it would fail when that was removed... But I guess this just defines the test, so agreed we should keep it for future use and may just need to update it down the road when we have another button in place.